riff-raff.yaml

```diff
@@ -4,7 +4,7 @@ deployments:
   viewer:
     type: autoscaling
     parameters:
-      bucket: composer-dist
+      bucketSsmLookup: true
```

Comment:
```suggestion
```
this can actually be removed entirely, as it's the default https://github.com/guardian/riff-raff/pull/885

Reply:
Thanks @twrichards!